using-mobile-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -258,6 +258,260 @@ ACPCore.collectLaunchInfo(userInfo)
+### setLogLevel
+
+#### Syntax
+
+```java
+public static void setLogLevel(LoggingMode mode)
+```
+
+#### Example
+
+```java
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+...
+
+MobileCore.setLogLevel(LoggingMode.VERBOSE);
+```
+
+{% endtab %}
+
+{% tab title="Objective-C" %}
+
+#### Objective-C
+
+### setLogLevel
+
+#### Syntax
+
+```objective-c
++ (void) setLogLevel: (ACPMobileLogLevel) logLevel;
+```
+
+#### Example
+
+```objective-c
+#import "ACPCore.h"
+...
+
+[ACPCore setLogLevel: ACPMobileLogLevelVerbose];
+```
+
+{% endtab %}
+
+{% tab title="Swift" %}
+
+#### Swift
+
+### setLogLevel
+
+#### Syntax
+
+```swift
++ (void) setLogLevel: (ACPMobileLogLevel) logLevel;
+```
+
+#### Example
+
+```swift
+import ACPCore
+...
+
+ACPCore.setLogLevel(ACPMobileLogLevel.verbose);
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+
+
+{% tabs %}
+
+{% tab title="Android" %}
+
+#### Java
+
+### getLogLevel
+
+#### Syntax
+
+```java
+public static LoggingMode getLogLevel()
+```
+
+#### Example
+
+```java
+LoggingMode mode = MobileCore.getLogLevel();
+```
+
+{% endtab %}
+
+{% tab title="Objective-C" %}
+
+#### Objective-C
+
+### getLogLevel
+
+#### Syntax
+
+```objective-c
++ (ACPMobileLogLevel) logLevel;
+```
+
+#### Example
+
+```
+var logLevel:ACPMobileLogLevel = [ACPCore logLevel];
+```
+
+{% endtab %}
+
+{% tab title="Swift" %}
+
+#### Swift
+
+### getLogLevel
+
+#### Syntax
+
+```swift
++ (ACPMobileLogLevel) logLevel;
+```
+
+#### Example
+
+```swift
+let logLevel:ACPMobileLogLevel = ACPCore.logLevel();
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+
+
+{% tabs %}
+
+{% tab title="Android" %}
+
+#### Java
+
+The `MobileCore` logging APIs use the `android.util.Log` APIs to log messages to Android. Based on the `LoggingMode` passed to `MobileCore.log()`, the following Android method is called:
+
+- `LoggingMode.VERBOSE` uses `android.util.Log.v`
+- `LoggingMode.DEBUG` uses `android.util.Log.d`
+- `LoggingMode.WARNING` uses `android.util.Log.w`
+- `LoggingMode.ERROR` uses `android.util.Log.e`
+
+All log messages from the Adobe Experience SDK to Android use the same log tag of ***AdobeExperienceSDK***. For example, if logging an error message using `MobileCore.log()`, the call to `android.util.Log.e` looks like `Log.e("AdobeExperienceSDK", tag + " - " + message)`.
+
+### log
+
+#### Syntax
+
+```java
+public static void log(final LoggingMode mode, final String tag, final String message)
+```
+
+#### Example
+
+```java
+MobileCore.log(LoggingMode.DEBUG, "MyClassName", "Provided data was null");
+```
+
+#### Output Example
+
+```
+D/AdobeExperienceSDK: MyClassName - Provided data was null
+```
+
+{% endtab %}
+
+{% tab title="Objective-C" %}
+
+#### Objective-C
+
+All log messages from the Adobe Experience SDK to iOS use a common format which contains the tag ***AdobeExperienceSDK***. For example, if logging an error message using `ACPCore.log()`, the printed output looks like `[AdobeExperienceSDK ERROR <tag>]: message`.
```

Comment:
- Please add a comma between "format" and "which"

- Please reword to: 

"The log messages from the Adobe Experience SDKs are printed to the iOS logging system/log file and use a common format that contains the **AdobeExperienceSDK** tag."